MapboxNavigation.xcodeproj/xcshareddata/xcschemes/MapboxCoreNavigation.xcscheme

```diff
@@ -60,6 +60,9 @@
                <Test
                   Identifier = "BridgingTests/testOfflineRouting">
                </Test>
+               <Test
+                  Identifier = "FreeDriveLocationManagerTests/testFreeDrive()">
+               </Test>
```

Comment:
I’ve disabled the free-driving integration test, because we don’t currently have a way to mock out the routing tile endpoint.

Reply:
#2546